tests/README.md

```diff
@@ -20,6 +23,25 @@ bazel test //tests --compilation_mode=dbg --test_output=errors --jobs=4 --runs_p
 
 `--jobs=4` is useful and is sometimes required to prevent too many processes to use GPU memory and cause CUDA out of memory issues.
 
+Additionally, accuracy tests are supported for Python backend using Nox. Please refer [dist-accuracy-test.sh](../docker/dist-accuracy-test.sh) for reference.
```

Comment:
Change `dist-accuracy-test.sh` file name if you changed it above.

Reply:
Done